Documentation/ceph-csi-drivers.md

```diff
@@ -116,7 +116,14 @@ expects a pool named `rbd` in your Ceph cluster. You can create this pool using
 [rook pool
 CRD](https://github.com/rook/rook/blob/master/Documentation/ceph-pool-crd.md).
 
-Please update `monitors` to reflect the Ceph monitors.
+Update the value of the `clusterID` field to match the ID of your
```

Comment:
As mentioned below, I don't believe we need the clusterID. The namespace of the ceph cluster will be sufficient.

Reply:
The key is called "clusterID" by csi. Currently I use the cluster UUID for maximum distinction, but we could use the namespace as a value, but the key cannot change.

Reply:
I'd suggest we follow the pattern we already have for rook storage classes as much as possible where we use the namespace as the identifier. Then they don't need to run a crazy command to get the cluster fsid. But since the csi driver is used outside rook i can see why it wouldn't make sense to rename the property to `clusterNamespace`. We'll just need to be clear in the examples that the `clusterID` is the namespace.

Reply:
OK, I can buy that argument. I'll make sure the docs are changed to say something like "When ceph csi is deployed by rook use the rook namespace (such as "rook-ceph")"

Reply:
done